src/coreclr/inc/enum_class_flags.h

```diff
@@ -49,4 +49,10 @@ inline auto operator ^= (T& left, T right) -> const decltype(T::support_use_as_f
     return left;
 }
 
+template <typename T>
+inline bool HasFlag(T value, decltype(T::support_use_as_flags) flag)
```

Comment:
```suggestion
bool HasFlag(T value, decltype(T::support_use_as_flags) flag)
```

The `inline` keyword shouldn't be needed in this case.